lib/src/screens/failure_page.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:io';
+
+import 'package:cake_wallet/generated/i18n.dart';
+import 'package:cake_wallet/src/widgets/primary_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_mailer/flutter_mailer.dart';
+import 'package:path_provider/path_provider.dart';
+
+class FailurePage extends StatelessWidget {
+  final String? error;
+  final StackTrace? stackTrace;
+
+  FailurePage({Key? key, this.error, this.stackTrace});
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    return Scaffold(
+      backgroundColor: Colors.grey.shade400,
+      body: Center(
+        child: Padding(
+          padding: EdgeInsets.symmetric(
+              horizontal: MediaQuery.of(context).size.width * 0.2),
+          child: Column(
+            mainAxisSize: MainAxisSize.min,
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Icon(
+                Icons.warning,
+                color: theme.errorColor,
+                size: 50,
+              ),
+              Padding(
+                padding: const EdgeInsets.symmetric(vertical: 20),
+                child: Text(
+                  "Oops, we got some error.",
+                  style: theme.textTheme.headline1?.copyWith(fontSize: 20),
+                ),
+              ),
+              Text(
+                "Please send crash report to our support team to make the application better.",
```

Comment:
Please send the crash report to our support team to make the application better.

(add `the`)

Reply:
this page was removed!! can you please make sure you are reviewing the latest commit

Reply:
but yeah the wording it self would be fixed